.github/workflows/build_macos.yml

```diff
@@ -22,9 +24,19 @@ jobs:
         env:
           IDF_PATH: ${{github.workspace}}/third_party/esp-idf
         run: make tools
-      - uses: actions/upload-artifact@v2
+      - name: Pack artifacts
+        run: |
+          tar -czf build/darwin.tgz -C ./build -s /^host/toit/ host
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: darwin-build
+          path: build/darwin.tgz
+      - name: Upload release artifacts
+        if: github.event_name == 'release'
+        uses: svenstaro/upload-release-action@v2
         with:
-          name: macos-build
-          path: |
-            build/host/bin/toit*
-            build/host/bin/lib/
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: build/darwin.tgz
```

Comment:
Should this be macos.tgz?

Reply:
I choose Darwin because that is how golang does but I'm not sure if that's the right way